### PR TITLE
add sin to tensorflow

### PR DESCRIPTION
**Pull request by PatricYan**
- [ ] #3352

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 1]
```diff
@@ -24,3 +24,7 @@ def cosh(x, name="cosh"):
 
 def fill(dims, value, name="full"):
     return ivy.full(dims, value)
+
+
+def sin(x):
```
Comment (rush2406) [inline ivy/functional/frontends/tensorflow/raw_ops.py:29]: can you please add `name="sin"`?

**Review by rush2406 — CHANGES_REQUESTED** [review]
Hi @PatricYan 
Thanks for contributing, there's a minor change required as mentioned below and please fix the merge conflicts as well!

File: ivy/functional/frontends/tensorflow/raw_ops.py (hunk 1 above)
Comment (PatricYan) [reply ivy/functional/frontends/tensorflow/raw_ops.py]: done

**Comment by PatricYan** [comment]
> 

Hi @rush2406 , I have resolved it by adding "name"

**Review by rush2406 — APPROVED** [review]

**Pull request merged**